ApollosProject/apollos-prototype
# Highlights the active card in HorizontalContentFeeds

## DESCRIPTION
<img src="https://user-images.githubusercontent.com/2053547/66954256-0a17bb00-f02e-11e9-8d91-35163d132a0c.png" width="30%" /><img src="https://user-images.githubusercontent.com/2053547/66954417-5f53cc80-f02e-11e9-96d9-184db426ea4d.png" width="30%" />

### What does this PR do, or why is it needed?
Highlights the active card in HorizontalContentFeeds so it is visually distinguishable from its siblings. 

### What design trade-offs/decisions were made?
None.

### How do I test this PR?
Tap into a content item with children in the app. Tap into a child scroll to the bottom to see it highlighted in the list with its siblings. 

### What automated tests did you write?
Saddly there is no test coverage over this component 😢 
## TODO:

- [x] Closes #1114 

## REVIEW:

**Manual QA**


**Code Review: Questions to consider**


> The purpose of PR Review is to _improve the quality of the software._
